contentcuration/contentcuration/static/js/edit_channel/views.js

```diff
@@ -137,13 +136,12 @@ var BaseView = Backbone.View.extend({
 		$(".first_focus_item").select();
 	},
 	set_indices: function(){
-        var selector = (this.el.id)? "#" + this.el.id : "." + this.el.className;
-        $(selector + " .tab_item").each(function(){
+        // var selector = (this.el.id)? "#" + this.el.id : "." + this.el.className;
+        $(this).select(".tab_item").each(function(){
```

Comment:
The most idiomatic way of doing this is `this.$el(".tab_item").each(...`

Convenience jquery wrapper of the DOM (http://backbonejs.org/#View-$el) should always be present.